.circleci/config.yml

```diff
@@ -108,9 +114,11 @@ workflows:
               only: master
           requires:
             - test
+            - acceptance
```

Comment:
_I think_ we can also re-use the dummy "build" step here, but looks like it's using the hokusai image and I am not sure if we plan to actually build stuff.